servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Single.java

```diff
@@ -1893,6 +1892,21 @@ public final Cancellable subscribe(Consumer<? super T> resultConsumer) {
         return subscriber;
     }
 
+    /**
+     * Subscribe to this {@link Single}, emit the result or error to one of the passed {@link Consumer}s.
+     *
+     * @param resultConsumer {@link Consumer} to accept the result of this {@link Single} if it succeeds.
+     * @param errorConsumer {@link Consumer} to accept the error of this {@link Single} if it fails.
+     * @return {@link Cancellable} used to invoke {@link Cancellable#cancel()} on the parameter of
+     * {@link Subscriber#onSubscribe(Cancellable)} for this {@link Single}.
+     */
+    public final Cancellable subscribe(Consumer<? super T> resultConsumer,
+                                       Consumer<? super Throwable> errorConsumer) {
```

Comment:
Do you want to enforce that this helper doesn't receive a null `errorConsumer`? If not, we might as well funnel the `subscribe(Consumer<? super T>)` variant through this method.
The same question applies to `Completable`.

Reply:
Good idea, changed ctors to use `requireNonNull` when we expect `errorConsumer`